src/scale/hierarchical.js

```diff
@@ -85,8 +90,7 @@ const HierarchicalScale = Scale.extend({
     // max 5 levels for now
     const ratios = [1, Math.pow(ratio, 1), Math.pow(ratio, 2), Math.pow(ratio, 3), Math.pow(ratio, 4)];
 
-    const distances = [];
-    {
+    const distances = []; {
```

Comment:
What is the `{ ... }` for? Just for code folding? The formatting looks a bit strange.